portafly/src/tests/components/AppLayout.test.tsx

```diff
@@ -56,18 +56,19 @@ describe('AppLayout tests', () => {
     expect(getByLabelText('Global navigation')).toBeInTheDocument()
   })
 
-  it('should hide the sidebar when clicking the nav-toggle button', async () => {
-    const { getByLabelText, getByTestId } = renderAppLayout()
-    const navButton = getByLabelText('Global navigation')
+  it('should start with collapsed sidebar', async () => {
```

Comment:
It should start collapsed by default, because of mobile-first approach defined [here](https://github.com/3scale/porta/blob/master/portafly/src/components/AppLayout.tsx#L51)